Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java

```diff
@@ -543,6 +543,7 @@ public class ScryfallImageSupportCards {
             add("MKM"); // Murders at Karlov Manor
             add("MKC"); // Murders at Karlov Manor Commander
             add("CLU"); // Ravnica: Clue Edition
+            add("ANB"); // Arena Beginner Set
```

Comment:
need to fix merge conflict, better to put this in chronological order (scryfall lists a date of 2020-08-13)